pam/internal/adapter/model.go

```diff
@@ -433,6 +441,12 @@ func (m uiModel) View() string {
 		view.WriteString(debug)
 	}
 
+	if view.Len() > 0 && m.canGoBack() {
+		infoMessage := infoMsgStyle.Render(fmt.Sprintf("Press escape key to %s",
+			goBackLabel(m.previousStage())))
```

Comment:
Any change to the strings in the `goBackLabel` function will likely result in an incorrect message here. To avoid that, I would prefer to implement another `goBackLabel` function here which includes the `Press escape key to ...` part instead of reusing the existing `goBackLabel` function.

Reply:
Ah you mean to duplicate what is in `goBackLabel` now, so basically drop commit 6175eefffd5ad4dfdfea542aeb73e4520e2261a8 ?

Reply:
Well, I wouldn't do it for now since we match, in case we need to diverge we can do it, but it would be caught quickly by the integration tests, no?

Reply:
fine by me